### ISSUE TITLE
Crash on ARM code (obfuscated sample)

### ISSUE THREAD

**Issue by enovella** [cmatthewbrooks/r2kit#5]
```c
[0x00049cbc]> #!pipe python2 /home/edu/r2scripts/functoyara.py -n rule -a edu
Traceback (most recent call last):
  File "/home/edu/r2scripts/functoyara.py", line 235, in <module>
    rule.create(args.name,args.author)
  File "/home/edu/r2scripts/functoyara.py", line 64, in create
    rule = self.create_rule(name,author)
  File "/home/edu/r2scripts/functoyara.py", line 85, in create_rule
    rule += '        ' + self.format_comment_instructions()
  File "/home/edu/r2scripts/functoyara.py", line 153, in format_comment_instructions
    comment_instructions += '// ' + op['bytes']
KeyError: 'bytes'
[0x00049cbc]> pd 5
/ (fcn) sym.Java_works_se_appsolid_CommonUtil_Emulator 184
|   sym.Java_works_se_appsolid_CommonUtil_Emulator ();
|           ; var int local_6ch @ sp+0x6c
|           ; var int local_70h @ sp+0x70
..... // SKIPPED
|           0x00049cbc  ~   f0b5           push {r4, r5, r6, r7, lr}
|           ;-- aav.0x00049cbd:
|              ; UNKNOWN XREF from 0x00000ab8 (aav.0x00000ab8 + 0)
|           0x00049cbd      b5             unaligned
|           0x00049cbe      ffb0           sub sp, aav.0x000001fc
|           0x00049cc0      ffb0           sub sp, aav.0x000001fc
|           0x00049cc2      efb0           sub sp, aav.0x000001bc
[0x00049cbc]> ~

```

Sample: 
[sample.so.zip](https://github.com/cmatthewbrooks/r2kit/files/1734940/sample.so.zip)

**Comment by cmatthewbrooks** [comment]
I just started seeing this same error on x86 too. I'll dig into it soon but it seems using op.get would be a quick fix.

**Comment by cmatthewbrooks** [comment]
The sample file here generates other r2 issues for me after my last update, but this specific KeyError error was fixed in 37982712eaad98811f1a80e78751b2c4d722c539.
